compiler/expressions.h

```diff
@@ -33,50 +33,48 @@ struct svalue;
 
 class SC3ExpressionParser : public BaseExpressionParser
 {
-public:
-  int evaluate(value* lval) {
-    return hier14(lval);
-  }
+  public:
+    int evaluate(value *lval) {
+        return hier14(lval);
+    }
 
-private:
-  typedef int (SC3ExpressionParser::*HierFn)(value*);
+  private:
+    typedef int (SC3ExpressionParser::*HierFn)(value *);
 
-  int hier14(value* lval);
-  int hier13(value *lval);
-  int hier12(value *lval);
-  int hier11(value *lval);
-  int hier10(value *lval);
-  int hier9(value *lval);
-  int hier8(value *lval);
-  int hier7(value *lval);
-  int hier6(value *lval);
-  int hier5(value *lval);
-  int hier4(value *lval);
-  int hier3(value *lval);
-  int hier2(value *lval);
-  int hier1(value *lval1);
-  int plnge(int *opstr,int opoff,HierFn hier,value *lval,
-            const char *forcetag,int chkbitwise);
-  int plnge1(HierFn hier,value *lval);
-  void plnge2(void (*oper)(void),
-              HierFn hier, value *lval1,value *lval2);
-  int plnge_rel(int *opstr,int opoff,HierFn hier,value *lval);
-  void callfunction(symbol *sym, const svalue *implicitthis, value *lval_result, int matchparanthesis);
-  int primary(value *lval);
-  int skim(int *opstr,void (*testfunc)(int),int dropval,int endval,
-           HierFn hier, value *lval);
-  int parse_view_as(value* lval);
+    int hier14(value *lval);
+    int hier13(value *lval);
+    int hier12(value *lval);
+    int hier11(value *lval);
+    int hier10(value *lval);
+    int hier9(value *lval);
+    int hier8(value *lval);
+    int hier7(value *lval);
+    int hier6(value *lval);
+    int hier5(value *lval);
+    int hier4(value *lval);
+    int hier3(value *lval);
+    int hier2(value *lval);
+    int hier1(value *lval1);
+    int plnge(int *opstr, int opoff, HierFn hier, value *lval, const char *forcetag,
```

Comment:
I'm not sure what the deal is, but it seems like where the asterisk gets nestled by the formatter varies.  Here, it's with the variable name and in other places its with the type.

Example in `compiler/assember.h`: `void assemble(const char* outname, memfile_t* fin);`

Reply:
I think I have to forcefully disable this in the clang-format config.